pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -111,7 +111,9 @@ class ModelCheckpoint(Callback):
 
     """
 
-    CHECKPOINT_NAME_LAST = "last.ckpt"
+    CHECKPOINT_JOIN_CHAR = "-"
+    CHECKPOINT_NAME_LAST = "last"
+    CHECKPOINT_SUFFIX = "ckpt"
```

Comment:
maybe rather as it is the same what `os.path.splitext(...)[-1]` returns
```suggestion
    CHECKPOINT_SUFFIX = ".ckpt"
```